src/udev/udevadm-trigger.c

```diff
@@ -240,6 +255,8 @@ int trigger_main(int argc, char *argv[], void *userdata) {
                 ARG_NAME = 0x100,
                 ARG_PING,
                 ARG_UUID,
+                ARG_DBM,
+                ARG_DBNM,
```

Comment:
not a fan of needless abbreviations. just call this ARG_DATABASE_MATCH and ARG_DATABASE_NOMATCH.

Reply:
I picked those abbreviations because this way it doesn't kill the existing formatting here https://github.com/systemd/systemd/blob/e2bcff148d08409a10f87900e96ed4058f1d35e9/src/udev/udevadm-trigger.c#L275-L283
plus it is a meaningless magic anyways.